includes/class-amp-theme-support.php

```diff
@@ -2321,6 +2251,18 @@ public static function prepare_response( $response, $args = [] ) {
 			}
 		}
 
+		$enable_optimizer = array_key_exists( ConfigurationArgument::ENABLE_OPTIMIZER, $args )
+			? $args[ ConfigurationArgument::ENABLE_OPTIMIZER ]
+			: true;
+
+		$enable_optimizer = apply_filters( Filter::ENABLE_OPTIMIZER, $enable_optimizer );
+
+		if ( $enable_optimizer ) {
+			$errors = new Optimizer\ErrorCollection();
+			self::get_optimizer( $args )->optimizeDom( $dom, $errors );
+			// @todo Deal with $errors.
```

Comment:
What errors could they be?

Reply:
The objects you can find in the `Error\` subfolder/subnamespace, like `CannotInlineRuntimeCss`.

Reply:
Great. In a subsequent PR we could look at exposing the errors in an HTML comment or perhaps the admin bar.

Reply:
I've gone ahead and listed out any errors encountered in an HTML comment at the end of the `head`. See 7b37383ea5177d4a4bec3ea45206001b1ac6aa93.